src/applications/mhv/secure-messaging/sass/search.scss

```diff
@@ -1,20 +1,95 @@
 @import "~@department-of-veterans-affairs/formation/sass/shared-variables";
 
+
 .search-form {
   background-color: var(--color-gray-lightest);
   padding: 16px;
 
+  va-accordion {
+    margin-left: -16px;
+    width: 90%;
+    @media (max-width: $small-screen) {
+    width: 106%;
+    }
+  }
+
+  .filter-input-box-container  {
+    display: flex;
+    flex-wrap: nowrap;
+    width: 100%;
+  }
+
+  .basic-filter-button {
+    margin-top: -5px;
+  }
+  .filter-button {
+    width: 109px;
+    padding: 13px;
+    padding-left: 20px;
+    padding-right: 20px;
+
+    margin-left: 0;
+    border-radius: 0 5px 5px 0;
+  }
+  .filter-text-input{
+    width: 70%;
+    margin: 0px;
+  }
+
+  .filter-input-box {
+    margin: 0px;
+  }
+
+  .custom-filter-button {
+    @media (max-width: $small-screen) {
+      width: 80%;
+      }
+  }
+
+  .advanced-search-field {
+    margin-left: 11px;
+  }
+
+  va-button {
+    margin-left: 11px;
+  }
+
+  .filter-content {
+    padding: 10px;
+    border-bottom: 5px solid var(--color-gray-lightest);
+    margin-bottom: -15px;
+    margin-left: -20px;
+    margin-right: -20px;
+    margin-top: -40px;
+    background-color: var(--color-gray-lightest);
+  }
+  .headline-text {
+    margin-left: -20px;
+
+  }
+
+  h3 {
+    margin: 0 0 10px 0;
+  }
+
   .search-in-description {
     margin: 0 0 6px;
     @media (min-width: $medium-screen) {
       margin: 0 0 24px;
     }
   }
+  .filter-box{
+    margin-top: -30px;
+    margin-bottom: -20px;
+  }
+
```

Comment:
I saw that the input field was off center from the button.  
<img width="735" alt="image" src="https://user-images.githubusercontent.com/59583325/232526528-61512ebf-7c23-473f-b757-33c6069ee639.png">
Try adding some css to strictly align the input with the button

Reply:
Hey im not able to reproduce this. Are you using the review instance?